### ISSUE TITLE
Mistake in question format

### ISSUE THREAD

**Issue by joshred83** [gderiddershanghai/gt_test_prep#1]
The following question uses a multiple choice format and does not allow the selection of multiple answers - 

Which of the following steps are involved in the training loop of a neural network?

A) Hyperparameter tuning

B) Backward pass

C) Preprocessing data

D) Forward pass

E) Updating weights

The correct answer was ['B', 'D', 'E']

Explanation:

The forward pass is used to compute the network's predictions, the backward pass calculates the gradients, and the weights are updated using the calculated gradients.

You can review Packt Coursera PyTorch course, Training Loops

**Comment by gderiddershanghai** [comment]
Unfortunately enabling multi-select would require me to rewrite a lot of the code and questions :( Perhaps over the summer, but the point is for you to review by practicing questions, not necessarily to get all the questions correct.

Do notify me off mistakes in the questions as those take 10 seconds to update. Cheers!
